internal/ingress/controller/store/backend_ssl_test.go

```diff
@@ -1,223 +0,0 @@
-}
-
-/*
-func TestSyncSecret(t *testing.T) {
-	// prepare for test
-	dCrt, dKey, dCa, err := buildCrtKeyAndCA()
-	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
-	}
-
-	foos := []struct {
-		tn            string
-		secretName    string
-		Data          map[string][]byte
-		expectSuccess bool
-	}{
-		{"getPemCertificate_error", "default/foo_secret", map[string][]byte{api.TLSPrivateKeyKey: dKey}, false},
-		{"normal_test", "default/foo_secret", map[string][]byte{api.TLSCertKey: dCrt, api.TLSPrivateKeyKey: dKey, tlscaName: dCa}, true},
-	}
-
-	for _, foo := range foos {
-		t.Run(foo.tn, func(t *testing.T) {
-			ic := buildGenericControllerForBackendSSL()
-
-			// init secret for getPemCertificate
-			secret := buildSecretForBackendSSL()
-			secret.SetNamespace("default")
-			secret.SetName("foo_secret")
-			secret.Data = foo.Data
-			ic.listers.Secret.Add(secret)
-
-			key := "default/foo_secret"
-			// for add
-			ic.syncSecret(key)
-			if foo.expectSuccess {
-				// validate
-				_, exist := ic.sslCertTracker.Get(key)
-				if !exist {
-					t.Errorf("Failed to sync secret: %s", foo.secretName)
-				} else {
-					// for update
-					ic.syncSecret(key)
-				}
-			}
-		})
-	}
-}
-
-func TestGetPemCertificate(t *testing.T) {
-	// prepare
-	dCrt, dKey, dCa, err := buildCrtKeyAndCA()
-	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
-	}
-
-	foos := []struct {
-		tn         string
-		secretName string
-		Data       map[string][]byte
-		eErr       bool
-	}{
-		{"secret_not_exist", "default/foo_secret_not_exist", nil, true},
-		{"data_not_complete_all_not_exist", "default/foo_secret", map[string][]byte{}, true},
-		{"data_not_complete_TLSCertKey_not_exist", "default/foo_secret", map[string][]byte{api.TLSPrivateKeyKey: dKey, tlscaName: dCa}, false},
-		{"data_not_complete_TLSCertKeyAndCA_not_exist", "default/foo_secret", map[string][]byte{api.TLSPrivateKeyKey: dKey}, true},
-		{"data_not_complete_TLSPrivateKeyKey_not_exist", "default/foo_secret", map[string][]byte{api.TLSCertKey: dCrt, tlscaName: dCa}, false},
-		{"data_not_complete_TLSPrivateKeyKeyAndCA_not_exist", "default/foo_secret", map[string][]byte{api.TLSCertKey: dCrt}, true},
-		{"data_not_complete_CA_not_exist", "default/foo_secret", map[string][]byte{api.TLSCertKey: dCrt, api.TLSPrivateKeyKey: dKey}, false},
-		{"normal_test", "default/foo_secret", map[string][]byte{api.TLSCertKey: dCrt, api.TLSPrivateKeyKey: dKey, tlscaName: dCa}, false},
-	}
-
-	for _, foo := range foos {
-		t.Run(foo.tn, func(t *testing.T) {
-			ic := buildGenericControllerForBackendSSL()
-			secret := buildSecretForBackendSSL()
-			secret.Data = foo.Data
-			ic.listers.Secret.Add(secret)
-			sslCert, err := ic.getPemCertificate(foo.secretName)
-
-			if foo.eErr {
-				if err == nil {
-					t.Fatal("Expected error")
-				}
-			} else {
-				if err != nil {
-					t.Fatalf("Unexpected error: %v", err)
-				}
-
-				if sslCert == nil {
-					t.Error("Expected an ingress.SSLCert")
-				}
-			}
-		})
-	}
-}
-*/
```

Comment:
why is this test completely deleted?

Reply:
Because the test was commented and the helpers unused. Also, the tests were related to static mode.